AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -280,7 +291,7 @@ def draw_axon_diameter(img, path_prediction, pred_axon, pred_myelin):
 
     path_folder = path_prediction.parent
 
-    stats_array = get_axon_morphometrics(pred_axon, path_folder)
+    stats_array = get_axon_morphometrics(pred_axon, path_folder , circle_approx = circle_approx)
```

Comment:
please be careful with PEP8... it's the 3rd typo...
```suggestion
    stats_array = get_axon_morphometrics(pred_axon, path_folder, circle_approx=circle_approx)
```

Reply:
Resolved in 3ba42bc